docs/src/reference/builtins.md

```diff
@@ -0,0 +1,317 @@
+# Built-Ins
+
+Stack comes with quite a few built-in functions (called `intrinsics` internally). They are the core of Stack and provide baseline functinoality.
+
+### Lists vs Functions
+
+Unless explicitly stated in the description of a function, all mentions of the type `list` includes functions (e.g.: `(fn 2 2 +)`).
+
+## Arithmetic
+
+### Add (`+`)
+
+**Signature:** `([a: int] [b: int] -- int)`
+
+**Equivalent Rust:** `a + b`
+
+### Subtract (`-`)
+
+**Signature:** `([a: int] [b: int] -- int)`
+
+**Equivalent Rust:** `a - b`
+
+### Multiply (`*`)
+
+**Signature:** `([a: int] [b: int] -- int)`
+
+**Equivalent Rust:** `a * b`
+
+### Divide (`/`)
+
+**Signature:** `([a: int] [b: int] -- int)`
+
+**Equivalent Rust:** `a / b`
+
+### Remainder (`%`)
+
+**Signature:** `([a: int] [b: int] -- int)`
+
+**Equivalent Rust:** `a & b`
```

Comment:
Stack uses wrapping arithmetic whereas Rust has bounds checking on arithmetic in safe modes. I'm not sure if it necessarily matters for the purposes of these examples though?